src/app/screens/ConfirmPayment/index.tsx

```diff
@@ -79,13 +79,18 @@ function ConfirmPayment() {
 
     try {
       setLoading(true);
+      // TODO: move to api
       const response = await msg.request(
         "sendPayment",
         { paymentRequest: paymentRequest },
         {
           origin: navState.origin,
         }
       );
+      if (response.error) {
```

Comment:
Double check if that is needed? Is that really the case for existing connectors too, that no errors would be shown?

Reply:
There are multiple places where an error is returned, and the sendPayment action is wrapped in try/catch. I think we need this check, I don't understand how it worked before.